packages/headless/src/controllers/commerce/product-view/headless-product-view.ssr.ts

```diff
@@ -0,0 +1,43 @@
+import {CommerceEngine} from '../../../app/commerce-engine/commerce-engine';
+import {SharedControllerDefinitionWithoutProps} from '../../../app/commerce-ssr-engine/types/common';
+import {
+  buildController,
+  Controller,
+} from '../../controller/headless-controller';
+import {
+  buildProductView,
+  ProductView as BaseProductView,
+} from './headless-product-view';
+
+export interface ProductViewDefinition
+  extends SharedControllerDefinitionWithoutProps<ProductView> {}
+
+/**
+ * Defines a `ProductView` controller instance.
+ *
+ * This controller is stateless and does not implement a `subscribe` method,
+ * making it simpler but different from other controllers in the system.
+ * Its sole purpose is to log an `ec_productView` event.
```

Comment:
[Our docs](https://docs.coveo.com/en/relay/latest/usage/#sending-events-with-relay) use `ec.productView`
```suggestion
 * Its sole purpose is to log an `ec.productView` event.
```